src/AudacityApp.cpp

```diff
@@ -502,6 +503,7 @@ static void QuitAudacity(bool bForce)
    // termination when exiting is requested
    #if !defined(__WXMAC__)
    LogWindow::Destroy();
+   FrameStatisticsDialog::Destroy();
```

Comment:
So the FrameStatisticsDialog is a per-application non-modal dialog (not per-project)?